.github/workflows/clean-pr.yml

```diff
@@ -1,21 +1,25 @@
-name: remove published site from netlify
+name: clean-pr
 
 on:
   pull_request:
     types:
       - closed
 
 jobs:
-  remove-site-from-netlify:
-    name: build
-    runs-on: ubuntu-18.04
+  netlify-remove:
+    runs-on: ubuntu-20.04
     if: github.event.pull_request.head.repo.fork == false
     steps:
-      - uses: actions/checkout@v2
+      -
+        name: Checkout
+        uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-      - name: delete from netlify
-        uses: ./.github/actions/netlify-clean
+      -
+        name: Remove site from Netlify
+        uses: docker/bake-action@v2
         with:
-          netlify_token: ${{ secrets.NETLIFY_AUTH_TOKEN }}
-          site_name: "${{ github.repository }}/${{ github.head_ref }}"
+          targets: netlify-remove
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_NAME: ${{ github.repository }}/${{ github.head_ref }}
```

Comment:
Looking at the [remove published site from netlify](https://github.com/docker/docker.github.io/actions/workflows/clean-pr.yml?query=is%3Asuccess) workflow, I think we could remove it in the future if we only accept pull request from forks as we already discussed. And we could disable the auto publish feature from master branch on Netlify too, so we would not need this workflow at all:

![image](https://user-images.githubusercontent.com/1951866/169715749-e2893c7d-b89d-40dc-ab14-1ed3ac5597f1.png)

Reply:
> And we could disable the auto publish feature from master branch on Netlify too, so we would not need this workflow at all:

Is that publish needed to get a preview for the "update published from master" PRs, such as https://github.com/docker/docker.github.io/pull/15128?